src/plugin.js

```diff
@@ -351,6 +352,10 @@ const eme = function(options = {}) {
         }
       }
     },
+    // Pass a promise polyfill from the player options for IE support. If none
```

Comment:
we should document the Promise requirement in the README as well. Also, currying seems fine here as it'll reduce the burden on users of this lib who would care to just know the supported CDMs.